code/game/objects/items/toys.dm

```diff
@@ -1169,6 +1170,26 @@
 		user << "<span class='alert'>Nothing happens.</span>"
 
 /*
+ * Snowballs
+ */
+
+/obj/item/toy/snowball
+	name = "snowball"
+	desc = "A compact ball of snow. Good for throwing at people."
+	icon = 'icons/obj/toy.dmi'
+	icon_state = "snowball"
+	throwforce = 12 //pelt your enemies to death with lumps of snow

```

Comment:
It kinda fits with christmas presents, And it will be hilarious to see someone accidentally fuck someone up with it.